### ISSUE TITLE
Sample project not compiling / running tests

### ISSUE THREAD

**Issue by spiffytech** [kumarshantanu/lein-clr#15]
I'm trying to run the sample project generated with `lein new lein-clr clr_test`, and it looks like the clojure code isn't getting copied to the `target` directory.

```
$ lein clr test
Running ClojureCLR tests
true

Unhandled Exception:
System.Exception: No namespace: clr-test.core-test found
...
```

```

$ lein clr compile :all
Compiling clr-test.core to /usr/src/app/target/clr/binSystem.Exception: namespace 'clr-test.core' not found after loading '/clr_test/core'
...
```

**Comment by kumarshantanu** [comment]
Closed via #14 

Thanks!

**Comment by spiffytech** [comment]
Sorry, #14 was an unrelated pull request. It does not solve this issue for me.

**Comment by kumarshantanu** [comment]
Reopening the issue. Could you please share sample code to reproduce the issue?

**Comment by spiffytech** [comment]
The sample code is just the project template that lein-clr includes. Creating a project with `lein new lein-clr clr_test` and running the above commands triggered these errors for me.

**Comment by kumarshantanu** [comment]
Can you try a project name without underscore? Maybe use a dash instead of underscore?

**Comment by spiffytech** [comment]
Ah! I created a project `clrtest` and everything works fine. Perhaps the plugin could throw an error if the user tries to create / run a project with a name that .NET can't handle?

**Comment by kumarshantanu** [comment]
Good idea. Will try to fix it in some time, and of course a patch would be welcome.
